packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1260,9 +1260,13 @@ class SelectionRect {
 }
 
 /// An interface to receive granular information from [TextInput].
-///
-/// Mixin with [TextInputClient] to receive any default implementations from
-/// [TextInputClient].
+/// 
+/// This class is susceptible to non-essential breaking changes when not mixed
+/// with a class that does not already mixin [TextInputClient]. In that case this
+/// class throws a missing concrete implementation error. [TextInputClient]
```

Comment:
nit: "throws" sounds like the error happens at run time. maybe: 

This mixin by itself does not provide any default implementations for `TextInputClient` methods. If you see xxx error consider mixing in the `TextInputClient` mixin.